modular_zubbers/code/modules/mob/check_pulse.dm

```diff
@@ -0,0 +1,27 @@
+/mob/verb/check_pulse(atom/A as mob in view())
```

Comment:
Would be much better to tie this to mob/living/ and allow ghosts to just get all the information on examine. Which they technically already do as they have full HUDs